front/app/modules/commercial/native_surveys/admin/components/SurveyBuilderSettings/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+// styles
+import styled from 'styled-components';
+import { colors } from 'utils/styleUtils';
+
+// components
+import { Title, Box, stylingConsts } from '@citizenlab/cl2-component-library';
+import Button from 'components/UI/Button';
+import CloseIconButton from 'components/UI/CloseIconButton';
+
+// intl
+import messages from './messages';
+import { FormattedMessage } from 'utils/cl-intl';
+
+const StyledBox = styled(Box)`
+  box-shadow: -2px 0px 1px 0px rgba(0, 0, 0, 0.06);
+`;
+
+const StyledCloseIconButton = styled(CloseIconButton)`
+  position: absolute;
+  right: 8px;
+`;
+
+const SurveyBuilderSettings = () => (
+  <StyledBox
```

Comment:
I'm not sure what the plan is - but do we see this content builder style layout being used for anything else in the future (e.g. more general form creation)? Maybe we could make these layout components (settings, toolbox and toolbox items, etc.) more general somewhere so we wouldn't have to duplicate the code?

Reply:
It might be better to abstract those out at such a time. This is because it might be hard to anticipate such changes and if they will fit in anything we do now. There would still be a high likelihood of changing the code. I would use [YAGN](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it#:~:text=%22You%20aren't%20gonna%20need,need%20it%22%20(YAGNI).) in this case

Reply:
Ok for sure, sounds good :+1: